nipype/interfaces/io.py

```diff
@@ -32,6 +32,8 @@
 from nipype.utils.misc import human_order_sorted
 from nipype.external import six
 
+from nipype.utils.config import NipypeConfig
```

Comment:
use relative path here:

`from ... import config` 

and use config directly. no need to initialize `NipypeConfig`